docs/en/guide/README.md

```diff
@@ -0,0 +1,15 @@
+# Guide
+
+If you want to debug, develop or execute SkyWalking Infra E2E, The following documentation would guide you.
+
+## Compile
+
+* [How to build?](compile/How-To-Build.md)
+
+## Execute
```

Comment:
You missed how to use in the GHA.

Reply:
Have already been added to execute, should I need to separate a new file to descript?

Reply:
We need to rename all `execute`. This is a very rare word used in the document. In here, this simply means `set up`.

Reply:
Sorry, I missed that. Have fixed.